.env

```
@@ -1,3 +1,3 @@
-VITE_API_BASE_URL=http://185.52.53.253/shaikh_carpets/api
-MONGODB_URI=mongodb://185.52.53.253:27017/shaikh-carpets-gst
+VITE_API_BASE_URL=http://localhost:5001/api
+MONGODB_URI=mongodb://localhost:27017/gst_invoice_system
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MONGODB_URI key should go before the VITE_API_BASE_URL key

(UnorderedKey)